proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -319,3 +324,15 @@ message PayloadCuratedPluginCreated {
   // owner_name is the name of the owner of the plugin.
   string owner_name = 2;
 }
+
+message PayloadTokenCreated {
+  // owner_id is the id of the owner of the token.
+  string owner_id = 1;
+  // token_expiry_time is the expiry time of the token.
+  google.protobuf.Timestamp token_expiry_time = 2;
```

Comment:
Do we want to add the token id (so we can correspond created/deleted audit events)?

Reply:
The token id and name will be logged as the affected resource. Do we also want it logged in the payload?

Reply:
Nope - sorry missed that part.